README.md

```diff
@@ -32,46 +32,41 @@ make -j4
 And that should give you `rbkit-app/RBKitClient` executable, which you can
 go ahead and run.
 
-## Qt build issue on OSX Mavericks
+## OSX Yosmite  
+Currenty there are 2 ways to compile rbkit-client on mac, with Qt Creator and with cmake.
 
-Qt binary distribution compiles with `-stdlib=libstdc++` for std lib,
-where as Xcode 5 on Mavericks compiles with `-stdlib=libc++`. So when you
-are using libraries compiled outside of Qt in the app, the linker will not
-be able to resolve the symbols in the stdlib. Fix this by doing to following.
-Go to `Projects` tab in Qt and under `Build Steps`, expand the qmake step and
-look for `-spec` argument. You need to add a flag `QMAKE_MACOSX_DEPLOYMENT_TARGET = 10.9`
-in the `qmake.conf` file of that spec.
+1. Using Qt Creator
+   Double click the `rbkit-client/RbkitClient.pro` file to open it in the qt creator. In the left menu-bar click on `Debug` button and select `rbkit-app` under the Run field.  
+   Hit the Run button, after compilation the GUI should pop-up.
 
-For example:
+2. Using cmake
+   Running a shadow build, i.e building from a separate directory is very
+simple. Just create a build directory, say `~/rbkit-build`. Lets say
```

Comment:
this example is as it is, just replaced its position a bit. If we need a directory like `Desktop`, I will update it with it